roles/openshift_named_certificates/tasks/main.yml

```diff
@@ -70,9 +70,10 @@
   command: >
     openssl verify -CAfile /etc/origin/master/ca-bundle.crt "{{ named_certs_dir }}{{ item.certfile | basename }}"
   with_items: "{{ named_certificates }}"
-  ignore_errors: True
   register: ca_bundle_result
   when: named_certificates_ca_file_defined | length > 0
+  change_when: false
```

Comment:
```suggestion
  changed_when: false
```

Reply:
Yeah, that's typo - I will fix it, but I need to rerun tests first. I will squash the commits